lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -372,6 +372,21 @@ MDagPath ufeToDagPath(const Ufe::Path& ufePath)
     );
 }
 
+bool isMayaRootPath(const Ufe::Path& ufePath)
```

Comment:
Could be simply 

return (ufePath.runTimeId() == g_MayaRtid) && (ufePath.size() == 1);